### ISSUE TITLE
[Bug] Stuck on pending 

### ISSUE THREAD

**Issue by SlurpelDur** [inorichi/tachiyomi-extensions#2971]
### Device information
* Tachiyomi version: 1.0.1 (tachiyomiJ2K)
* Android version: 9

### Source information
* Source name:  Any Source 
* Source extension version:  Any Version

## Steps to reproduce
1. First Step: Open tachiyomiJ2K and select whatever source you want to download/update.
2. Second Step: Press the install/update button.

### Expected behavior
The source should have updated or installed without any issues.

### Actual behavior
The source is stuck on pending and will not download or update.

### Other details
I can still update and download sources through the extension list. 
                               ↓
https://tachiyomi.org/extensions/
---
I'm also on the tachiyomiJ2K app instead on the regular tachiyomi app.

**Comment by arkon** [comment]
This is an issue with the app, not the extensions. Do you have a VPN or ad blocker enabled?

**Comment by SlurpelDur** [comment]
I have lookout installed. I'm not sure if that's considered a VPN.

**Comment by arkon** [comment]
I don't know what that is, but have you tried turning it off?

**Comment by SlurpelDur** [comment]
No, I don't think it has a off button. I'll look and see if I can turn it off tho.

**Comment by SlurpelDur** [comment]
I still get the same issue with it off

**Comment by arkon** [comment]
Can you try installing stable 0.9.1 and seeing if it works there?

**Comment by SlurpelDur** [comment]
What's that?

**Comment by arkon** [comment]
The regular Tachiyomi app.

**Comment by SlurpelDur** [comment]
It's the same in the regular app, still stuck on pending.
